test/lib/deviceInterface.ts

```diff
@@ -0,0 +1,141 @@
+import { DeviceInterface as SNDeviceInterface } from '@standardnotes/snjs';
+import LocalStorage from './localStorage';
+
+const KEYCHAIN_STORAGE_KEY = 'keychain';
+
+/**
+ * The DeviceInterface implementation to handle storage and keychain operations.
+ */
+export default class DeviceInterface extends SNDeviceInterface {
```

Comment:
the naming is a bit confusing - I was hoping for an interface instead of a class judging by the name.

Reply:
This name comes from SNJS. Device interface as in "to interface with the device". But I can see how it can be mistaken for a code concept.